### PR TITLE
Update README.md

### LINKED ISSUE #2: docs: add proper documentation for README.md

### PR DESCRIPTION
**Pull request by nayakastha**
Merging this PR fixes #2

### CODE REVIEW

**Comment by iamdipanshusingh** [comment]
@nayakastha, check `Here's how you can start`... it is showing the link as is... and plz remove my profile pic for now :sweat_smile:

**Comment by iamdipanshusingh** [comment]
@nayakastha also... make the `What's API` section a bit shorter... remove the first para... keep it simple...

**Comment by nayakastha** [comment]
Check It out now.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-## 💡 What's Unique?
+
+# 💡 What's Unique?
 <br> The handling of APIs is usually messy. So here we present to you a much simplified way to beautifully handle the API calls.
 
-## Features
-- [x] Manage token 
+# Features
+ Here's the list of your all-in-one service
+- [ ] Manage token
```
Comment (iamdipanshusingh) [inline README.md]: @nayakastha you have removed the mark from her by mistake... plz revert it and you're good to go

Comment (nayakastha) [reply README.md]: I am so sorry. I am doing it right away

**Review by iamdipanshusingh — APPROVED** [review]
LGTM, will merge it soon!

**Comment by iamhssingh** [comment]
@iamdipanshusingh LMK if @nayakastha needs to be added as a contributor to the repository directly.

**Comment by iamdipanshusingh** [comment]
@nayakastha do let us know... if you want to contribute any further, we'll add you as contributor.. :slightly_smiling_face:

**Comment by nayakastha** [comment]
I would like to contribute surely.😃

**Comment by iamdipanshusingh** [comment]
@iamhssingh sir plz add her as a contributor... @nayakastha plz have a look at the issues list and you can tag me if needed.. have added some documentation and code related issues there... don't hesitate to add one yourself... and welcome to 101Loop :grinning:

**Pull request merged**